test/e2e/data/shared/v1beta1/metadata.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
   - major: 1
-    minor: 2
+    minor: 3
```

Comment:
Should this be updated? more curious than anything.

Reply:
yes, each time we have to update capi version, we have to update the actual URL path, the version number in the name in that file, and in the mtadat file. Otherwise, the e2e tests were failing to install CAPI in the test cluster.

Reply:
I will creaate. aseparate PR today to add a note to taht file